bluemira/codes/_freecadapi.py

```diff
@@ -1324,6 +1326,31 @@ def sweep_shape(profiles, path, solid=True, frenet=True):
         return solid_result.Shells[0]
 
 
+def fillet_wire_2D(wire: apiWire, radius: float, chamfer: bool = False) -> apiWire:
+    """
```

Comment:
Is it necessary to check that the wire is planar or is it up to the user?

Reply:
It does actually work for 3-D wires, but I dare not expose that just yet... I will put in a defense against non-planar wires